docs/README.md

```diff
@@ -0,0 +1,10 @@
+# Mapzen Terrain Tile documentation
+The [Mapzen terrain tiles](https://mapzen.com/projects/joerd) provide basemap terrain coverage of the world in a raster tile format.
```

Comment:
not sure if we're going to have a project page for terrain tiles yet, so holding off on this link